mikupad.html

```diff
@@ -2953,6 +3069,14 @@
 						{ name: 'Show on hover while holding CTRL', value: 1 },
 						{ name: 'Don\'t show', value: -1 },
 					]}/>
+				<div className="modal-width-adjustment">
+					<span>Box width: </span>
+					<button onClick=${() => adjustWidth(-5)}>⟪</button>
+					<button onClick=${() => adjustWidth(-1)}>⟨</button>
+					<input type="text" value="35" id="modal-width-readout" readOnly />
+					<button onClick=${() => adjustWidth( 1)}>〉</button>
+					<button onClick=${() => adjustWidth( 5)}>⟫</button>
+				</div>

```

Comment:
Please allow the manual change of the input value and use a react state as I commented before.
Also, a button you can optionally hold down is better than simply having another button for larger increments.